### PR TITLE
fix(card): toggle stretch mode

### PR DESCRIPTION
**Pull request by mistermalm**
## **Describe pull-request**  
Scaling cards

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [CDEP-3445-scaling-cards](https://tegel.atlassian.net/browse/CDEP-3445)    

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to storybook for Card
2. Toggle stretch on/off and see how the card changes

## **Checklist before submission**
- [ ] I have added unit tests for my changes (if applicable)
- [ ] All existing tests pass

## **Suggested test steps**
- [ ] Storybook controls

## **Additional context**  
Unit test might be added later

### CODE REVIEW

File: packages/core/src/components/card/card.stories.tsx [hunk 1]
```diff
@@ -110,18 +110,30 @@ export default {
...
     subheader: 'Subheader text',
     thumbnail: true,
     imagePlacement: 'Below',
     bodyImg: false,
-    bodyContent: '',
+    bodyContent: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.`,
     bodyDivider: false,
     cardActions: `<tds-icon slot="actions" size="20px" name="arrow_right"></tds-icon>`,
     clickable: false,
+    stretch: true,
```
Comment (timrombergjakobsson) [inline packages/core/src/components/card/card.stories.tsx:136]: One minor thing:
```suggestion
    stretch: false,
```

**Review by timrombergjakobsson — APPROVED** [review]
Nice!

File: packages/core/src/components/card/card.stories.tsx [hunk 2]
```diff
@@ -110,18 +110,30 @@ export default {
         defaultValue: { summary: false },
       },
     },
+    stretch: {
+      name: 'Stretch card body',
+      description: 'Toggles if the card body to take scale with the card.',
```
Comment (nathalielindqvist) [inline packages/core/src/components/card/card.stories.tsx:115]:
```suggestion
      description: 'Toggles if the card body should scale with the card.',
```

**Review by nathalielindqvist — APPROVED** [review]
Nice work! Apart from one small suggestion on the Storybook description it looks good to me! 🌟

**Pull request merged**